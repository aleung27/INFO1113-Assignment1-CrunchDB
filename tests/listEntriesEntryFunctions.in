SET a 9
LIST ENTRIES
PUSH a 0 1
LIST ENTRIES
APPEND a 1 2
LIST ENTRIES
PICK a 3
LIST ENTRIES
PLUCK a 1
LIST ENTRIES
POP a
LIST ENTRIES
POP a
POP a
POP a
POP a
LIST ENTRIES
