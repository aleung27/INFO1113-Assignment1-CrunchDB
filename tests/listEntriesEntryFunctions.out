> ok

> a [9]

> ok

> a [1 0 9]

> ok

> a [1 0 9 1 2]

> 9

> a [1 0 9 1 2]

> 1

> a [0 9 1 2]

> 0

> a [9 1 2]

> 9

> 1

> 2

> nil

> a []

